docs/development/building-from-sources.md

```diff
@@ -78,21 +84,52 @@ To build on Linux, you need:
 
 ```
 
-```{tab-item} MacOS
+```{tab-item} Linux with conda
 
-To build on MacOS, you need:
+In this setup you would install both Python and the necessary build requirements
+in a conda environment.
 
-- A working native compiler toolchain, enough to build
-  [CPython](https://devguide.python.org/getting-started/setup-building/index.html#macos-and-os-x).
+You would need a working native compiler toolchain, enough to build
+  [CPython](https://devguide.python.org/getting-started/setup-building/index.html#linux), for example,
+- `apt install build-essential` on Debian based systems.
+- Conda which can be installed from [MiniForge](https://github.com/conda-forge/miniforge)
+
+Then run,
+
+- `conda create -c conda-forge -n pyodide-env python=3.11`
+- `conda activate conda-forge`
+- `conda install -c conda-forge nodejs ccache f2c pkg-config swig make patch pkg-config texinfo autoconf automake libtool`
```

Comment:
If someone knows how to nest a non inline code block here let me know. The problem is that there are already twice nested
````
```
````
code blocks for creating tabs, and apparently one can't just recursively add more ` everywhere to make it work)

Reply:
See my comment below: https://github.com/pyodide/pyodide/pull/3644#discussion_r1130278271